rskj-core/src/main/java/co/rsk/peg/P2shErpRedeemScriptBuilder.java

```diff
@@ -0,0 +1,82 @@
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.Utils;
+import co.rsk.bitcoinj.core.VerificationException;
+import co.rsk.bitcoinj.script.Script;
+import co.rsk.bitcoinj.script.ScriptBuilder;
+import co.rsk.bitcoinj.script.ScriptOpCodes;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class P2shErpRedeemScriptBuilder implements ErpRedeemScriptBuilder{
+    private static final Logger logger = LoggerFactory.getLogger(P2shErpRedeemScriptBuilder.class);
+
+    public Script createRedeemScript(List<BtcECKey> defaultPublicKeys,
+                                     List<BtcECKey> emergencyPublicKeys,
+                                     long csvValue) {
+        Script defaultRedeemScript = ScriptBuilder.createRedeemScript(
+            defaultPublicKeys.size() / 2 + 1,
```

Comment:
Should we also receive the threshold as a parameter? It's actually defined in the specific Federation class `getNumberOfSignaturesRequired`

Reply:
Makes sense. The builder should just build, not decide things (like this number)

Reply:
And we should have two thresholds, one for emergency and one for default keys